addons/binding/org.openhab.binding.icloud/src/main/java/org/openhab/binding/icloud/handler/BridgeHandler.java

```diff
@@ -139,7 +139,7 @@ public Address getAddress(PointType location) {
             }
         } catch (Exception e) {
             logger.debug("getAddress failed:");
-            logger.debug(json);
+            logger.debug("{}", json);
             logException(e);

```

Comment:
You'd better combine those to one line. @martinvw your opinion?

Reply:
Yes, please combine it in a single statement: 

```java
logger.debug("getAddress failed: {}", json, e);
```